cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java

```diff
@@ -64,9 +64,10 @@ public CgmesRegulatingControlRatio buildRegulatingControlRatio(String id, String
         return rtc;
     }
 
-    public CgmesRegulatingControlPhase buildRegulatingControlPhase(String regulatingControlId,
+    public CgmesRegulatingControlPhase buildRegulatingControlPhase(String id, String regulatingControlId,
```

Comment:
I didn't find out where you added the regulatingControlId of transformers?

Reply:
In the `AbstractTransformerConversion.addCgmesReferences` method:

```java
if (tc.getRegulatingControlId() != null) {
    transformer.setProperty(cgmesReferenceKey(tc.getId(), "TapChangerControl"), tc.getRegulatingControlId());
}
```